.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -35,11 +35,23 @@
 
 ### Test Coverage
 
-<!-- TODO: Please show how you have added tests to cover your changes,
-  -        or tell us how you tested it. For each systems you tested,
-  -        uncomment the systems in the list below.
+<!-- TODO: Please do one of the following:
+  -    * Create unit tests, and explain here how they cover your changes.
+  -    * List steps you used for manual testing, and explain how they cover
+  -      your changes.
+  -      For each system you manually tested, uncomment it below.
   -->
 
+<!-- I created tests for ... -->
```

Comment:
I don't know what you expect me to write here, can you either update this to be more specific or remove it? And similar to the below comment, I think we want to minimize the amount of things a person deletes or ignores from the template so consider removing this unless it's adding a lot of value.